java/src/main/native/src/TableJni.cpp

```diff
@@ -1469,7 +1470,8 @@ JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_TableWithMeta_releaseTable(JNIE
 
 JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_Table_readJSONFromDataSource(
     JNIEnv *env, jclass, jobjectArray col_names, jintArray j_types, jintArray j_scales,
-    jboolean day_first, jboolean lines, jboolean recover_with_null, jlong ds_handle) {
+    jboolean day_first, jboolean lines, jboolean recover_with_null, jboolean mixed_types_as_string,
```

Comment:
New mixed_types_as_string argument is unused?

Reply:
Thanks. missed it. Nice catch.

Reply:
updated one of the tests.